src/org/usfirst/frc4904/sovereignty/FusedSensor.java

```diff
@@ -0,0 +1,50 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+public class FusedSensor<T> implements Fusible<T> {
```

Comment:
As we talked about, for extensibility and usability it makes sense to keep this as a generic and leave the output type up to the implementing sensor. While attempting to demonstrate my solution to Yoni, which initially used doubles, I chose to implement a button as a use case. It became immediately clear that the output should be variable to prevent unnecessary manual casting or conversion.